apps/main/src/entities/gauge/model/query-options.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file entities/gauge/model.ts
+ * @description This file defines the data fetching and mutation logic for gauge-related operations in the Curve.fi DApp.
+ * It's a core part of the 'gauge' entity in the FSD architecture.
+ *
+ * This module exports functions that create React Query options for various gauge operations
```

Comment:
These functions are very specific to tanstack query and are only used once in the query hook. Doesn't it make sense to move them directly to the hooks file?